main.js

```diff
@@ -0,0 +1,26 @@
+var newList = function(listName) {
+  this.id = "id-" + listName;
```

Comment:
This actually won't handle uniqueness -- if I create two Lists with the same name they'll have the same ID! We need some way to either:

1. Stop a user from creating a new list if one already exists with the same name.
1. Keep a global count so that each List is different.